deps/rabbitmq_prometheus/src/collectors/prometheus_rabbitmq_core_metrics_collector.erl

```diff
@@ -414,6 +428,8 @@ label(M) when is_map(M) ->
               end, <<>>, M);
 label(#resource{virtual_host = VHost, kind = exchange, name = Name}) ->
     <<"vhost=\"", VHost/binary, "\",exchange=\"", Name/binary, "\"">>;
+label({#resource{virtual_host = VHost, kind = queue, name = Name}, P, _}) when is_pid(P) ->
+      <<"vhost=\"", VHost/binary, "\",queue=\"", Name/binary, "\"","channel=\"",(iolist_to_binary(pid_to_list(P)))/binary, "\"">>;
```

Comment:
recently another PR https://github.com/rabbitmq/rabbitmq-server/pull/9656 was merged which adds escaping to label values. I think you need to rebase to latest main and add escaping to this line as well (the vhost and queue name values)

Reply:
thanks @gomoripeti I did completely miss this!